packages/atomic/cypress/integration/result-list/folded-result-list.cypress.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  FoldedResultListSelectors,
+  getAtomicResultAtIndex,
+} from './result-list-selectors';
+import {generateComponentHTML, TestFixture} from '../../fixtures/test-fixture';
+import {
+  addFoldedResultList,
+  buildTemplateWithoutSections,
+} from './result-list-actions';
+import {makeChildComponents} from './folded-result-list-utils';
+
+const setSource = () => {
+  cy.intercept({method: 'POST', path: '**/rest/search/v2**'}, (request) => {
+    request.body.aq = '@source==("iNaturalistTaxons")';
+  });
+};
+
+describe('Folded Result List Component', () => {
+  it('should show child results', () => {
+    new TestFixture()
+      .with(setSource)
+      .with(
+        addFoldedResultList(
+          buildTemplateWithoutSections([
+            generateComponentHTML('atomic-result-link'),
+            makeChildComponents(),
+          ])
+        )
+      )
+      .init();
+
+    FoldedResultListSelectors.firstResult()
+      .contains('Pyrenomycetes')
+      .should('be.visible');
+
+    getAtomicResultAtIndex(FoldedResultListSelectors, 0)
+      .find('atomic-result-link')
+      .contains('Stubble Lichens');
+    getAtomicResultAtIndex(FoldedResultListSelectors, 1)
+      .find('atomic-result-link')
+      .contains('Opercs');
+  });
+
+  it('should show grandchild results', () => {
+    new TestFixture()
+      .with(setSource)
+      .with(
+        addFoldedResultList(
+          buildTemplateWithoutSections([
+            generateComponentHTML('atomic-result-link'),
+            makeChildComponents(makeChildComponents()),
+          ])
+        )
+      )
+      .init();
+
+    FoldedResultListSelectors.firstResult()
+      .contains('Pyrenomycetes')
+      .should('be.visible');
+
+    getAtomicResultAtIndex(FoldedResultListSelectors, 0)
+      .find('atomic-result-link')
```

Comment:
can we have all selectors in a separate file? so it's will be easy to manage them in the future

Reply:
@lvu285 can you please let me know if my latest commit is good enough?